lib/Conch/Route.pm

```diff
@@ -54,38 +54,143 @@ sub all_routes (
     $root->add_type(uuid => Conch::UUID::UUID_FORMAT);
 
 
-    # GET /doc
+
+=head3 C</doc>
+
+Returns API docs.
+
+=cut
+
+   # GET /doc
     $root->get('/doc', sub ($c) { $c->reply->static('public/doc/index.html') });
 
-    # GET /ping
+
+=head3 C</ping>
+
+Returns a json document C<{ "status" : "ok" }> for use in health checks from
+monitoring systems and potentially load balancers.
+
+=cut
+
+   # GET /ping
     $root->get('/ping', sub ($c) { $c->status(200, { status => 'ok' }) });
 
-    # GET /version
+
+=head3 C</version>
+
+Returns the current version/tag for the deploy of the server in a json document
+like C<{ "version":"0.001" }>.
+
+=cut
+
+   # GET /version
     $root->get('/version', sub ($c) { $c->status(200, { version => $c->version_tag }) });
 
+
+=head3 C<POST /login>
+
+Handles the act of logging in.
+
+=over 4
+
+=item Request Schema: L<Login|https://conch.joyent.us/schema/request/Login>
+
+=item Response Schema: L<Login|https://conch.joyent.us/schema/response/Login>
+
+=back
+
+=cut
+
     # POST /login
     $root->post('/login')->to('login#session_login');
 
+
+=head3 C<POST /logout>
+
+Logs a user out by expiring their session. Returns C<204 NO CONTENT>.upon
+success.
+
+=cut
+
     # POST /logout
     $root->post('/logout')->to('login#session_logout');
 
+=head3 C<POST /reset_password>
+
+Deprecated. Returns a C<410 GONE>.
+
+=cut
+
     # POST /reset_password -> GONE
     $root->post('/reset_password', sub ($c) { $c->status(410) });
 
-    # GET /schema/:input_or_response/:schema_name
+=head3 C<GET /schema/:input_or_response/:schema_name>
+
+=over 4
+
+=item * Response Schema: L<draft-07 JSON-Schema|http://json-schema.org/draft-07/schema>
```

Comment:
I think I've made this more clear. I documented in the copy that they're links to the request and response schema (with links to the overall schema files). I also split out the URLs to be more explicit rather than leaning on `:input_or_response`.